### DISCUSSION TITLE
Autolaunch

### DISCUSSION THREAD

**Discussion by sskit** [lllyasviel/stable-diffusion-webui-forge#1415] [General]
Hello, please tell me how to disable automatic opening of the page in the browser.

--disable-in-browser & --nobrowser - does not work

**Comment by aggregate15** [comment]
Go to Settings > From Left bar, choose "System" and under "Automatically open webui in browser on startup" choose Disabled. Apply settings

![Untitled](https://github.com/user-attachments/assets/e0256048-cf15-457f-86a4-512749cebd89)
